.github/workflows/test.yml

```
@@ -0,0 +1,30 @@
+      - uses: software-mansion/setup-scarb@v1
+        with:
+          scarb-version: 2.11.4
+      - name: Check cairo format
+        run: scarb fmt --check
+      - name: Build cairo programs
+        run: scarb build
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)